src/lib/coaler/embedder/ConformerEmbedder.cpp

```diff
@@ -29,14 +30,16 @@ namespace coaler::embedder {
         auto matches = RDKit::SubstructMatch(*mol, *m_core, matchParams);
         assert(!matches.empty());
 
-        for (auto const& match : matches) {
+        for (auto const &match: matches) {
             CoreAtomMapping molQueryCoords;
-            for (const auto& [queryId, molId] : match) {
+            for (const auto &[queryId, molId]: match) {
                 molQueryCoords[molId] = m_coords.at(queryId);
             }
 
             // embed molecule conformers
             RDKit::DGeomHelpers::EmbedParameters params;
+            params = RDKit::DGeomHelpers::ETKDGv3;
```

Comment:
I guess we should use `srETKDGv3` as this appears to be an optimized version

Reply:
`srETKDGv3()` just adds SmallRingTorsion if I understand correctly
```
   # this includes addtional small ring torsion potentials
   params = AllChem.srETKDGv3()

   # this includes additional macrocycle ring torsion potentials and macrocycle-specific handles
   params = AllChem.ETKDGv3()

   # to use the two in conjunction, do:
   params = AllChem.ETKDGv3()
   params.useSmallRingTorsions = True

   # a macrocycle attached to a small ring
   mol = Chem.MolFromSmiles("C(OC(CCCCCCC(OCCSC(CCCCCC1)=O)=O)OCCSC1=O)N1CCOCC1")
   mol = Chem.AddHs(mol)
   AllChem.EmbedMultipleConfs(mol, numConfs = 3 , params = params)
```
From [this](https://github.com/rdkit/rdkit/blob/9c1d1a84f4798b23cba54daf9739e5e6cb2c9b29/Docs/Book/Cookbook.rst#L2427)

Reply:
Resolved in 7fc13ef